### DISCUSSION TITLE
Relocating files to the wrong folder

### DISCUSSION THREAD

**Discussion by Irrel22** [seanap/Plex-Audiobook-Guide#66] [Q&A] [answered]
Hello! I don't know why I'm having issues with the relocate files action but no matter what path I try Mp3Tag either makes a folder inside my Temp folder or the Programs folder where it is installed. Any idea what I have messed up?

![Mp3tag_aPSXsnRI0C](https://user-images.githubusercontent.com/100799521/156434068-56a2ad90-8b0b-4329-a69a-11edc84a43fb.png)

Thanks!

**Comment by seanap** [comment, marked as answer]
I would double check all folder locations used in the Action.  You can edit the *.mta action files directly like I outline in the guide or go into the actions menu in mp3tag and click the edit icon

![image](https://user-images.githubusercontent.com/17012946/156435983-2d48a085-ddce-45b9-adfc-f66e369dfa98.png)

**Reply by Irrel22** [reply]
That fixed it! Thank you.
